MultiServer.py

```diff
@@ -2254,12 +2254,15 @@ async def main(args: argparse.Namespace):
                 if not isinstance(e, ImportError):
                     logging.error(f"Failed to load tkinter ({e})")
                 logging.info("Pass a multidata filename on command line to run headless.")
-                exit(1)
+                # when cx_Freeze'd the built-in exit is not available, so we import sys.exit instead
```

Comment:
Wondering if it's worth reporting this as a bug? sys.exit is easy to forget when exit works in non-frozen.

Reply:
Done in https://github.com/marcelotduarte/cx_Freeze/issues/1937